dataGen.js

```diff
@@ -7,70 +7,70 @@ const tickerChars = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
 
 const companies = new Set();
 
-const numToseed = 1000000;
+const numToseed = 100000;
 
 
 const mapTicker = () => {
-    let company = '';
-    for (let i = 0; i <= 4; i++) {
-      company += tickerChars.charAt(Math.floor(Math.random() * tickerChars.length));
-    }
-    if (!companies.has(company)) {
-      if (company === undefined) {
-        return assignCompanies();
-      }
-      companies.add(company);
-      return company;
-    }
-    return mapTicker();
-  };
-  
-  const createFakeStock = () => ({
-    ask_price: faker.finance.amount(100, 1500, 6),
-    ask_size: faker.random.number({ min: 100, max: 500 }),
-    bid_price: faker.finance.amount(100, 2000, 6),
-    bid_size: faker.random.number({ min: 100, max: 500 }),
-    last_extended_hours_trade_price: faker.finance.amount(100, 2000, 6),
-    last_trade_price: faker.finance.amount(100, 2000, 6),
-    symbol: mapTicker(),
-    quantity: faker.finance.amount(1, 500, 4),
-  });
-  
-  const generateStocks = () => {
-    return createFakeStock();
-  };
-  
-  const convertArrayofObjectsToCSV = (data, count) => {
-    let result = '';
-    if (count === numToseed) {
-      result += 'ask_price,ask_size,bid_price,bid_size,last_extended_hours_trade_price,last_trade_price,symbol,quantity,createdAt,updatedAt\n';
-    }
-    for (var property in data) {
-      result += data[property] + ',';
+  let company = '';
+  for (let i = 0; i <= 4; i++) {
+    company += tickerChars.charAt(Math.floor(Math.random() * tickerChars.length));
+  }
+  if (!companies.has(company)) {
```

Comment:
Set is definitely a faster lookup than an Array. The code is pretty [readable](https://en.wikipedia.org/wiki/Computer_programming#Readability_of_source_code)/clearly written too. If optimizing further for time complexity, consider looking into an alternative to [Math.random()](https://stackoverflow.com/questions/18501998/is-it-faster-to-shuffle-a-list-or-generate-it-randomly).